src/prognostic_equations/zero_velocity.jl

```diff
@@ -18,7 +39,29 @@ function zero_velocity_tendency!(Yₜ, Y, p, t)
     return nothing
 end
 
-# Turn off all momentum tendency derivatives in the advection test.
+"""
+    zero_velocity_jacobian!(∂Yₜ_err_∂Y, Y, p, t)
+
+Modifies the Jacobian matrix `∂Yₜ_err_∂Y` for an "advection test" scenario
+(`p.atmos.advection_test == true`).
+
+For matrix rows corresponding to velocity variables (grid-mean `uₕ`, `u₃`, and
+EDMFX updraft `u₃ʲ` if applicable):
+- Diagonal blocks (e.g., `∂(uₕ_tendency)/∂uₕ`) are set to represent `-I` (negative identity).
+- Off-diagonal blocks (e.g., `∂(uₕ_tendency)/∂(tracer)`) are set to zero.
+
+This effectively decouples the velocity evolution from other variables in the
+linearized system used by an implicit solver, or simplifies their implicit
+contribution to behave like a strong relaxation to zero tendency (if the Jacobian
+is for $Y' - \Delta t J Y'$ and $J = -I/\Delta t$). This is useful if velocities
```

Comment:
```suggestion
is for `Y' - \Delta t J Y'` and `J = -I/\Delta t`). This is useful if velocities
```
using `$` like that in a doc string is attempting to escape the doc string, can we change it to tick marks?